contentcuration/contentcuration/frontend/RecommendedResourceCard/components/RecommendedResourceCard.vue

```diff
@@ -43,37 +49,67 @@
 
 <script>
 
+  import find from 'lodash/find';
+  import LearningActivities from 'kolibri-constants/labels/LearningActivities';
+  import { mapState } from 'vuex';
+  import { ContentKindLearningActivityDefaults } from 'shared/leUtils/ContentKinds';
+  import Checkbox from 'shared/views/form/Checkbox';
+  import ContentNodeLearningActivityIcon from 'shared/views/ContentNodeLearningActivityIcon';
+
   export default {
     name: 'RecommendedResourceCard',
+    components: {
+      Checkbox,
+      ContentNodeLearningActivityIcon,
+    },
     props: {
-      to: {
+      node: {
         type: Object,
         required: true,
       },
-      title: {
-        type: String,
-        default: null,
+    },
+    computed: {
+      ...mapState('importFromChannels', ['selected']),
+      channelName() {
+        //Todo: We'll most likely not have null channel names, so remove check after integration
+        return this.node.channel?.name || '';
+      },
+      learningActivities() {
+        return {
+          [ContentKindLearningActivityDefaults[this.node.kind] || LearningActivities.EXPLORE]: true,
```

Comment:
This should use the actual learning activities. See the following URL for a response from the public endpoint. https://studio.learningequality.org/api/public/v2/contentnode/1aec7435425a521b968f35851acb6de3/